core/src/main/java/org/infinispan/remoting/inboundhandler/GlobalInboundInvocationHandler.java

```diff
@@ -154,6 +154,7 @@ private ReplicableCommandRunner(ReplicableCommand command, Reply reply,
       @Override
       public void run() {
          try {
+            // TODO can we remove this?
```

Comment:
not sure what is the state about custom commands (or modules command).

Reply:
I think we can move it so that it's in the `else` block of the `command instanceof GlobalRpcCommand`, because if it's a `GlobalRpcCommand` (new interface as of 11) it should be utilising the component registry passed with `invokeAsync`. @danberindei Any thoughts?

Reply:
+1 to move it.